docs/source/config.rst

```diff
@@ -426,3 +426,19 @@ Miscellaneous
    Ignore any restart files and repeat the initial run upon resubmission. This
    is generally only used for testing purposes, such as bit reproducibility.
 
+``modules``
+   Specify a list of environment modules to load at the start of the PBS job,
+   for example::
+
+      modules:
+         - netcdf-c-4.9.0-intel-2021.6.0-t5vmcra
+         - parallel-netcdf-1.12.3-intel-2021.6.0-xty6sc5
+         - xerces-c-3.2.3-intel-2021.6.0-z57iwgl
```

Comment:
I think it would be better to have more "generic" module names in here. The `spack` ones look quite strange to most people.

Reply:
```suggestion
         - netcdf-c-4.9.0
         - parallel-netcdf-1.12.3
         - xerces-c-3.2.3-intel
```